src/main/java/org/sonarlint/intellij/util/SonarLintUtils.java

```diff
@@ -46,6 +46,8 @@
 import java.util.regex.Pattern;
 import javax.annotation.Nullable;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.jps.model.java.JavaModuleSourceRootTypes;
```

Comment:
Good catch. It does not work on PHPStorm. And the worst is that there is no runtime error/logging happening, it simply does not work.